.travis.yml

```diff
@@ -13,8 +13,14 @@ matrix:
         - php: 5.4
         - php: 5.5
         - php: 5.6
+        - php: 5.6
+          env: SYMFONY_VERSION="^2.1"
         - php: 7.0
           env: CHECKS=yes
+        - php: 7.1
+          env: SYMFONY_VERSION="^3.0"
+        - php: 7.2
+          env: SYMFONY_VERSION="^4.0@dev"
```

Comment:
just `^4.0` is fine

Reply:
You asked in #247 to "ensure Travis will cover 2.x, 3.x and 4.x", maybe I misunderstood?

Reply:
I meant
```diff
-env: SYMFONY_VERSION="^4.0@dev"
+env: SYMFONY_VERSION="^4.0"
```